frontends/mit-learn/src/pages/HomePage/HeroSearch.tsx

```diff
@@ -185,50 +187,36 @@ const HeroSearch: React.FC = () => {
         </Typography>
         <ControlsContainer>
           <SearchInput
-            placeholder="Search for courses, programs, learning, and teaching materials"
+            placeholder="Search for courses, programs, and learning materials..."
             size="hero"
             fullWidth
             value={searchText}
             onChange={onSearchChange}
             onClear={onSearchClear}
             onSubmit={onSearchSubmit}
           />
-          <LinksContainer>
-            <TrenderingContainer>
-              <Typography
-                sx={{ marginRight: "8px" }}
-                typography={{ xs: "subtitle4", md: "subtitle3" }}
-              >
-                Trending
-              </Typography>
+          <>
```

Comment:
**Requested Change:** The *styling* on `LinksContainer` was unncessary, but having a container element was necessary. I.e., replace `LinksContainer` with a `div`. Removing it entirely makes the topic links pick up undesirable behavior from the flex parent.